lablink-client-base/lablink-client-service/lablink_client_service/update_inuse_status.py

```diff
@@ -0,0 +1,118 @@
+import time
+import logging
+import requests
+import os
+
+try:
+    import psutil
+except ImportError as e:
+    print("psutil is not installed. Please install it using 'pip install psutil'.")
```

Comment:
why would psutil not be installed in client package? shouldn't it always be installed?